### ISSUE TITLE
 | (0/100, 0 it/hr) [0s:0s]2023/07/11 23:53:53 open /home/vinayak/mosint-config.json: no such file or directory

### ISSUE THREAD

**Issue by vinayak220296** [alpkeskin/mosint#73]
| (0/100, 0 it/hr) [0s:0s]2023/07/11 23:53:53 open /home/vinayak/mosint-config.json: no such file or directory 
 how to resolve this

**Comment by minzak** [comment]
Same issues.

If I make empty this file i got:
```

unexpected end of JSON inputunexpected end of JSON input
Social Media Results:
|- Spotify [Not here!]
|- Adobe [Not here!]
|- Twitter [Not here!]
|- Discord [Not here!]
|- Instagram [Not here!]
unexpected end of JSON inputunexpected end of JSON inputunexpected end of JSON input
```

**Comment by SumithFernandes** [comment]
open /home/dell/mosint-config.json: no such file or directory

Need assistance to resolve this issue.

**Comment by SumithFernandes** [comment]
I have checked all files are set while manually checking the path open/home/dell/ there is already a file named mosint-config.json
 Again run go command but the response is same below 
open /home/dell/mosint-config.json: no such file or directory

Seeking assistance from anybody who knows actual reason and solution please.

**Comment by zake1god** [comment]
023/07/24 05:15:04 open /home/zake/mosint-config.json: no such file or directory
need help

**Comment by Habibu-R-ahman** [comment]
> | (0/100, 0 it/hr) [0s:0s]2023/07/11 23:53:53 open /home/vinayak/mosint-config.json: no such file or directory how to resolve this

Create a file `/home/vinayak/mosint-config.json`

```
{
 "Breachdirectory": "",
 "Emailrep": "",
 "Hunter": "",
 "Intelx": "",
 "Psbdmp": ""
}

```

**Comment by alpkeskin** [comment]
Bugs have been fixed in the new version (v3.0.0). Ensure the config file is configured correctly for **Mosint** to work properly.
